pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -79,7 +80,8 @@ const (
 
 	// AWS Defaults
 	defaultAWSCredentialsSecret = "aws-cloud-credentials"
-	defaultAWSInstanceType      = "m4.large"
+	defaultAWSX86InstanceType   = "m5.large"
```

Comment:
i just want to make sure we highlight this change as it might have impact on users who were expecting the smaller type machines.

Reply:
Also newer processors.  m5 appears to be slightly cheaper at this point but I suppose that could vary by region.

I'm willing to break this out into a separate PR if you think it's a good idea.

Reply:
i'm ok keeping it in here, just wanted to make sure we have good visibility.

Reply:
cc @JoelSpeed @enxebre

Reply:
This value was added to be inline with the defaults that the installer uses. If they have updated to the newer instance types, we should too, if they haven't, I'd prefer to keep with what they use

Reply:
https://github.com/openshift/installer/commit/51f66d63cf96007856863d1c126dacc6a5ddafc4

Reply:
> i'm ok keeping it in here, just wanted to make sure we have good visibility.

Unless you want to bump to m5 in 4.8 and target the ARM changes for 4.9 (iow post 4.8 branchpoint)?